tests/core/vm/test_sharding_vm.py

```diff
@@ -7,26 +12,64 @@
     new_sharding_transaction,
 )
 from tests.core.vm.contract_fixture import (
-    contract_bytecode,
-    contract_address,
+    simple_transfer_contract_bytecode,
+    simple_transfer_contract_address,
+    simple_contract_factory_bytecode,
+    CREATE2_contract_bytecode,
+    CREATE2_contract_address,
 )
 
 
 def test_sharding_transaction(shard_chain_without_block_validation):  # noqa: F811
```

Comment:
Since it's a test in `test_vm_state.py`, mind updating the test name to `test_sharding_apply_transaction`?

Reply:
Sure. I will update the name. Do you mean also moving it to `test_vm_state.py`?

Reply:
ohhhhh sorry, I meant `test_sharding_vm.py`!